tests/admin_views/test_adminsite.py

```diff
@@ -43,6 +54,13 @@ def test_each_context(self):
         self.assertEqual(ctx["site_url"], "/")
         self.assertIs(ctx["has_permission"], True)
 
+    def test_custom_admin_titles(self):
+        request = self.request_factory.get(reverse("test_custom_adminsite:index"))
+        request.user = self.u1
+        ctx = custom_site.each_context(request)
+        self.assertEqual(ctx["site_title"], custom_site.site_title)
+        self.assertEqual(ctx["site_header"], custom_site.site_header)
```

Comment:
I'd prefer to assert against concrete values:
```suggestion
        self.assertEqual(ctx["site_title"], "Custom title")
        self.assertEqual(ctx["site_header"], "Custom site")
```
For example, the current version will not fail when removing `site_title` and `site_header` from `CustomAdminSite`.

Reply:
Makes sense. Updated the PR.